### ISSUE TITLE
Drone build failed due to "manifest unknown"

### ISSUE THREAD

**Issue by aiyengar2** [rancher/rancher#30067]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

Failed to push a Drone image tag for `quay.io/prometheus-operator/prometheus-config-reloader` and `quay.io/prometheus-operator/prometheus-operator`. This has worked successfully for a prior tag that has been pushed via the scripts (e.g. `v0.43.0`) but failed for `v0.39.0`.

Seems like the error is due to an unknown manifest, but it could also be related to pushing an image tag that is older than the latest pushed image tag.

```
Line: quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.39.0
--
855 | quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.39.0
856 | time="2020-11-12T23:37:02Z" level=fatal msg="Error parsing image name \"docker://quay.io/prometheus-operator/prometheus-config-reloader:v0.39.0\": Error reading manifest v0.39.0 in quay.io/prometheus-operator/prometheus-config-reloader: manifest unknown: manifest unknown"
857 | ===
858 | Failed copying image for rancher/coreos-prometheus-config-reloader
859 | ===
860 | quay.io/prometheus-operator/prometheus-config-reloader:v0.39.0 has unknown schemaVersion
861 | ===
862 | Failed copying image for rancher/coreos-prometheus-config-reloader
863 | ===
864 |  
865 | Line: quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.39.0
866 | quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.39.0
867 | time="2020-11-12T23:37:03Z" level=fatal msg="Error parsing image name \"docker://quay.io/prometheus-operator/prometheus-operator:v0.39.0\": Error reading manifest v0.39.0 in quay.io/prometheus-operator/prometheus-operator: manifest unknown: manifest unknown"
868 | ===
869 | Failed copying image for rancher/coreos-prometheus-operator
870 | ===
871 | quay.io/prometheus-operator/prometheus-operator:v0.39.0 has unknown schemaVersion
872 | ===
873 | Failed copying image for rancher/coreos-prometheus-operator
874 | ===
```

Drone build: https://drone-publish.rancher.io/rancher/image-mirror/50/1/2

**Comment by aiyengar2** [comment]
Need a resolution for this to unblock https://github.com/rancher/rancher/issues/29149

**Comment by aiyengar2** [comment]
Both repositories `quay.io/prometheus-operator/prometheus-config-reloader` and `quay.io/prometheus-operator/prometheus-operator` do not have a tag for `v0.39.0` since Prometheus Operator never migrated to using `quay.io/prometheus-operator/*` until release `v0.38.3` based on the [release notes](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.38.3). `v0.38.3` was released after `v0.39.0`.


We should be pulling images from the old repository at `quay.io/coreos/*` instead, which has both image tags:
- [`quay.io/coreos/prometheus-config-reloader:v0.39.0`](https://quay.io/repository/coreos/prometheus-config-reloader?tag=v0.39.0&tab=tags)
- [``quay.io/coreos/prometheus-operator:v0.39.0`](https://quay.io/repository/coreos/prometheus-operator?tag=v0.39.0&tab=tags)
